js/tiles.js

```diff
@@ -2451,7 +2451,7 @@ const TileMesh = {
         createImages: function() {
             this.wallTop = this.createImage("tiles/rooms/wall/top.svg");
             this.wallRight = this.createImage("tiles/rooms/wall/rightwaves.svg");
-            this.ground = this.createImage("tiles/rooms/floor/bubbleMesh.svg");
+            this.ground = this.createImage("tiles/rooms/floor/landscape.svg");
```

Comment:
Why do you remove a tile to add a new one?

Reply:
```suggestion
            this.ground = this.createImage("tiles/rooms/floor/landscape.svg");
```
I have removed the tile "bubbleMesh.svg" because it was not my own tile. That's why I have added my own tile "landscape.svg" in place of that.

Reply:
now please review it

Reply:
Also bubbleMesh.svg was already added so I decided to add something new